nemo/collections/common/tokenizers/huggingface/auto_tokenizer.py

```diff
@@ -173,6 +173,10 @@ def add_special_tokens(self, special_tokens_dict: dict) -> int:
             setattr(self, k, getattr(self.tokenizer, k, None))
         return num_tokens_added
 
+    @property
+    def additional_special_tokens_ids(self):
```

Comment:
Do we need `additional_special_tokens` when we already have `special_tokens` ?

Reply:
`special_tokens` is never assigned as an attribute of this class right?

Reply:
Can we just add a docstring explaining what additional_special_tokens are?